x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/resource_findings/resource_findings_container.tsx

```diff
@@ -86,10 +87,18 @@ const getResourceFindingSharedValues = (sharedValues: {
     }),
     description: sharedValues.clusterId,
   },
+  {
+    title: i18n.translate('xpack.csp.findings.resourceFindingsSharedValues.cloudAccountName', {
+      defaultMessage: 'Cloud Account Name',
+    }),
+    description: sharedValues.cloudAccountName,
+  },
```

Comment:
Why do we need an additional check and filter? If we are already filtering truthy  below:

`const getModifiedTitlesListItems = (listItems: EuiDescriptionListProps['listItems']) =>
  listItems
    ?.filter((item) => !!item?.title && !!item?.description)
    .map((item) => ({ ...item, title: `${item.title}:` }));`
    
   I only see Cloud Account Name for CSPM integration and Cluster Id for KSPM resources.

Reply:
Totally forgot we even had this function, the way `getModifiedTitlesListItems` is written looks more like a safety measure to make sure we don't render titles without data. In general, I wouldn't pass empty data on purpose just because somewhere down the line "someone else" is filtering out empty data. 
I'm approving anyway cause it does work as intended, sorry for holding this one back.
Regarding the suggested change, I think its a better approach but it's nitpicking at this stage so it will be your call